### DISCUSSION TITLE
Pensando backlog de tareas

### DISCUSSION THREAD

**Discussion by emrodk** [Godot-en-Espanol/Node-City#5] [General]
* Menues
  * Inicio: 4 botones... Jugar, Opciones, Créditos y Salir
    *  Jugar: Carga escena pertinente al inicio de gameplay
    * Opciones: Permite configurar cosas básicas como volumen (música y sfx por separado), resolución y pantalla completa. (algo más?)
    * Créditos: se entiende
    * Salir: se entiende
  * Pausa
    * Volver
    * Opciones
    * Salir
  * Game Over
    * Reinicio Nivel
    * Menu principal
    * Cargar partida (? si se guarda) 
  * Pantalla de Victoria
    * Puntaje: Score... tanques destruidos
  * Configurar de inputs
* Gameplay
  * Mapa/Escenario
    * (Extra) Generador procedural de mapa 
    * Editor de mapa (in game)
    * Contenido general:
      * Tipos de bloques / tiles y sus efectos
      * Puntos de spawn para tanques e items
      * Base del Player
  * Oleada de enemigos #14   
  * Editor de uso interno para audio (tool experimental) :
    * Poder controlar buses, volumen, pitch, etc
    * Poder escuchar señales o leer data relevante de juego
  * Tanques #11 
  * PowerUps #16
  * Disparo #9 
  * IA enemigos #11 
    * (en proceso) el tanque básico es drunk walk, el tanqué rápido va directo a la bandera, El tanque pesado sigue al jugador, etc.
* Extras
  * Sistema de guardado (A espera de mecánicas)
  * Algo más?

**Comment by ultrazar** [comment]
Faltaría también hacer el sistema de guardado, no?

**Reply by emrodk** [reply]
no la había pensado,habría que pensar que podría guardar y que valga la pena. Ahora creo un post para debatir esto mismo
[link al post](https://github.com/Godot-en-Espanol/Node-City/discussions/6)

**Comment by gfazzito** [comment]
Se podría agregar un menu de Gameover con los siguientes botones:
* reinicio nivel
* menú principal
* Cargar partida

**Reply by emrodk** [reply]
agregado
